examples/todo/ui/todo.slint

```diff
@@ -50,6 +53,10 @@ MainWindow := Window {
         { title: "Profit", checked: false },
     ];
 
+    property <bool> is-header-visible: false;
+    property <bool> is-sort-by-name: false;
+    property <bool> is-filter-done: false;
```

Comment:
This reads as if the property indicates if the "filter" is finished (done). I suggest to rename this to "filter-done-items" or "hide-done-items" (like the label).